kafka-ansible/roles/kafka/tasks/main.yml

```diff
@@ -10,7 +10,13 @@
   - "{{ kafka_run_dir }}"
 
 - name: create kafka data directory
-  file: path={{ item }} state=directory mode=0755
```

Comment:
Please remove `name: Set deploy_user - set ansible_user as default` task or set `delpoy_user` in all.yml, see https://github.com/pingcap/tidb-ansible/blob/master/group_vars/all.yml#L4.

https://github.com/pingcap/thirdparty-ops/search?q=deploy_user&unscoped_q=deploy_user